tests/Browser/ManageAccountsTest.php

```diff
@@ -38,10 +40,14 @@ public function testRoleDropdown()
         });
     }
 
+    /**
+     * @group manage
+     * @return void
+     */
     public function testModalAppearsAfterRoleChange()
```

Comment:
Deze test werkt nog niet. Ik krijg de volgende error:

 element not interactable
  (Session info: chrome=123.0.6312.122)

  at vendor\php-webdriver\webdriver\lib\Exception\WebDriverException.php:98
     94▕             switch ($status_code) {
     95▕                 case 'element click intercepted':
     96▕                     throw new ElementClickInterceptedException($message, $results);
     97▕                 case 'element not interactable':
  ➜  98▕                     throw new ElementNotInteractableException($message, $results);
     99▕                 case 'insecure certificate':
    100▕                     throw new InsecureCertificateException($message, $results);
    101▕                 case 'invalid argument':
    102▕                     throw new InvalidArgumentException($message, $results);

  1   vendor\php-webdriver\webdriver\lib\Exception\WebDriverException.php:98
  2   vendor\php-webdriver\webdriver\lib\Remote\HttpCommandExecutor.php:359

Daarintegen doet de andere test van deze klasse het wel weer!

Reply:
Ik snap er echt helemaal niks meer van, bij mij slaagt die test wel gewoon. Misschien werkt het als je eerst `php artisan dusk:install` doet...?

Reply:
Nope, dat was het niet :( ik krijg nog steeds dezelfde error

Reply:
Deze error krijg ik ook